File: src/main/java/dev/rafi/vitelist/utils/ChatUtils.java
package dev.rafi.vitelist.utils;

import com.velocitypowered.api.command.CommandSource;
import net.kyori.adventure.text.Component;

public class ChatUtils {

    public static Component colorize(String msg) {
        return Component.text(msg.replace('&', '§'));
    }

    public static void sendMessage(CommandSource source, String msg) {
        source.sendMessage(colorize(msg));
    }
}

File: src/main/kotlin/dev/rafi/vitelist/utils/ChatUtils.kt
package dev.rafi.vitelist.utils

import com.velocitypowered.api.command.CommandSource
import net.kyori.adventure.text.Component


fun colorize(msg: String) : Component {
    return Component.text(msg.replace('&', '§'))
}
fun sendMessage(source: CommandSource, msg: String) {
    source.sendMessage(colorize(msg))
}